src/NodaTime/CalendarOrdinal.cs

```diff
@@ -11,30 +11,24 @@ namespace NodaTime
     internal enum CalendarOrdinal
     {
         Iso = 0,
-        Gregorian1 = 1,
-        Gregorian2 = 2,
-        Gregorian3 = 3,
-        Gregorian4 = 4,
-        Gregorian5 = 5,
-        Gregorian6 = 6,
-        Gregorian7 = 7,
-        Julian = 8,
-        Coptic = 9,
-        HebrewCivil = 10,
-        HebrewScriptural = 11,
-        PersianSimple = 12,
-        PersianArithmetic = 13,
-        PersianAstronomical = 14,
-        IslamicAstronomicalBase15 = 15,
-        IslamicAstronomicalBase16 = 16,
-        IslamicAstronomicalIndian = 17,
-        IslamicAstronomicalHabashAlHasib = 18,
-        IslamicCivilBase15 = 19,
-        IslamicCivilBase16 = 20,
-        IslamicCivilIndian = 21,
-        IslamicCivilHabashAlHasib = 22,
-        UmAlQura = 23,
+        Gregorian = 1,
+        Julian = 2,
```

Comment:
Exactly. It changes the serialized form, but not in a way that impacts any currently-serialized data. (And I think we're going to break serialization between 1.3 and 2.0 anyway, for the sake of sanity.)